crates/lib/tests/rules.rs

```diff
@@ -115,6 +115,20 @@ fn main() {
 
                 continue;
             }
+
+            let template = case
+                .configs
+                .get("core")
+                .and_then(|it| it.as_map())
+                .and_then(|it| it.get("templater"))
+                .and_then(|it| it.as_string());
+            if let Some(template) = template {
+                if template == "jinja" {
+                    println!(" ignored, jinja templating is not supported");
+                    continue;
+                }
```

Comment:
Since we don't support any templating engines, it might make sense to ignore all of them.

Reply:
👍